entities/card.py

```diff
@@ -1,32 +1,44 @@
 from dataclasses import dataclass
-from entities.card_type import CardType
+
+from flask import Request
+
 from entities.card_design_properties import CardDesignProperties
-from datetime import datetime
-from uuid import UUID
-from typing import Dict
-from card_type import CardType
+from typing import Dict, List
+from entities.card_proprties import CardProperties
+from consts import *
 
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
 
 
 @dataclass
 class Card:
-    content: str
-    type: CardType
+    card_properties: CardProperties
     design_properties: CardDesignProperties
-    date: datetime
-    version: str
-    id: UUID
 
-    def serialize_card(self) -> Dict[str, str]:
+    def card_to_dict(self) -> Dict[str, str]:
         serialized_card = {
-            "content": self.content,
-            "card_type": self.card_type.value,
-            "date": str(self.date),
-            "version": self.version,
-            "id": self.id
+            CONTENT: self.card_properties.content,
+            CARD_TYPE: self.card_properties.type.value,
+            DATE: str(self.card_properties.date),
+            VERSION: self.card_properties.version,
+            ID: self.card_properties.id
         }
         for key, value in serialized_card.items():
             if value is None:
-                serialized_card[key] = "null"
+                serialized_card[key] = NULL
 
         return serialized_card
+
+    @staticmethod
+    def cards_to_dict(cards: List['Card']) -> CARDS_SERIALIZED:
+        cards_dict = {
+            CARDS: []
+        }
+        for card in cards:
+            cards_dict[CARDS].append(card.card_to_dict())
+        return cards_dict
+
+    @staticmethod
+    def generate_card(request: Request) -> 'Card':
```

Comment:
Why do this function get reqeust?
Card shouldn't know it's generated by http request, it shouldn't know how it's generated ( I think it breaks the D in SOLID but don't count on it, I just don't think it's a correct design). 
Replace it to get a json string or something of a sort.